tests/dags/test_logging_in_dag.py

```diff
@@ -25,6 +25,11 @@
 
 
 def test_logging_fn(**kwargs):
+    """
+    Tests DAG logging.
+    :param kwargs:
+    :return:
+    """
```

Comment:
Is this related?

Reply:
Yep. I had to fix it. Pylint is not 100% deterministic and if you analyze it all together it detects some errors that were not reported before.